en/ros/ros2_offboard_control.md

```diff
@@ -187,22 +176,19 @@ void OffboardControl::publish_vehicle_command(uint16_t command, float param1,
 	msg.source_system = 1;
 	msg.source_component = 1;
 	msg.from_external = true;
-
+	msg.timestamp = this->get_clock()->now().nanoseconds() / 1000;
 	vehicle_command_publisher_->publish(msg);
 }
 ```
 
-As the description suggests, the above code serves the purpose of sending `VehicleCommand` messages with commands to the flight controller.
-
-## Usage
-
-After building the colcon workspace, and after starting PX4 SITL (`make px4_sitl_rtps gazebo-classic`, which starts the microRTPS client automatically on UDP ports 2019 and 2020) and the microRTPS agent (`micrortps_agent -t UDP`, starting the agent connected to UDP ports 2020 and 2019):
+:::note
+[VehicleCommand](../msg_docs/VehicleCommand.md) is one of the simplest and most powerful ways to command PX4, and by subscribing to [VehicleCommandAck](../msg_docs/VehicleCommandAck.md) you can also confirm that setting a particular command was successful.
+The param and command fields map to [MAVLink commands](https://mavlink.io/en/messages/common.html#mav_commands) and their parameter values.
+:::
 
-```sh
-$ source path_to_colcon_workspace/install/setup.bash
-$ ros2 run px4_ros_com offboard_control
-```
+<!--
 
 ## Demo with PX4 SITL and Gazebo Classic
 
 @[youtube](https://youtu.be/Nbc7fzxFlYo)
+-->
```

Comment:
MY VM does not have the grunt to run the simulation and record. So if someone wants to create a video and upload would be very pleased!